### DISCUSSION TITLE
IOS framework integrated filament run error "dyld[34221]: Symbol not found in flat namespace '_UBERARCHIVE_DEFAULT_OFFSET'"

### DISCUSSION THREAD

**Discussion by guoaobao** [google/filament#6114] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17892094/192775425-ab11cc9f-51f4-4383-862c-ccf221fee993.png)
![image](https://user-images.githubusercontent.com/17892094/192776488-57321309-df52-473e-9495-10dc4f83dea9.png)
![image](https://user-images.githubusercontent.com/17892094/192776571-33980325-6219-4ad1-b681-0132ae3966f5.png)

IOS framework integrated filament run error "dyld[34221]: Symbol not found in flat namespace '_UBERARCHIVE_DEFAULT_OFFSET'",how to solve?

**Comment by bejado** [comment, marked as answer]
Are you compiling Filament yourself? You need to make sure you're linking against the `libuberarchive.a` library.

**Reply by guoaobao** [reply]
thank you!
